lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -133,5 +142,5 @@ def load_config(config_data):
 
     if 'region' not in config_data['aws_lambda']:
         raise Exception('"region" is mandatory under the "aws_lambda" or "aws" section of the configuration')
-    elif 'region' not in config_data['aws']:
-        config_data['aws']['region'] = config_data['aws_lambda']['region']
+    elif 'region' not in config_data['aws_lambda']['aws']:
+        config_data['aws_lambda']['aws']['region'] = config_data['aws_lambda']['region']
```

Comment:
Hi Josep, now I understand... I'll add some comments to the code to explain this. What I was thinking is that all AWS configuration is inherited from the standard AWS SDK ways, but I understand that is also important to enforce users deploy lambdas and buckets in the same region to avoid data transfer costs. In this case, the safest option is require to specifiy the "region" parameter the "aws" section, regardless of the AWS SDK config, and not have it repeated in "aws_s3" or "aws_lambda"?